src/gameComponents/DefenseTokens/DefenseToken.java

```diff
@@ -0,0 +1,152 @@
+package gameComponents.DefenseTokens;
+
+import engine.GameConstants;
+
+public class DefenseToken {
+    /**
+    A defense Token is a token that can be assigned to squadrons or ships. A defense token has a variety of states
+    The Defense token class allows for the creation of defense tokens and allows for the various state changes
+    it may undergo during normal use
+     */
+
+    //The type of token
+    protected GameConstants.defenseTokenType type;
```

Comment:
Any compelling reason for `protected` over `private`?  In general its best practice to restrict access as much as you can get away with.

Reply:
no? Just muscle memory from college in 4 years ago. protected as I remembered it made it accessible within the package (Which admittedly is itself) but not outside the package. I forget why we used protected when we did, using it just made me feel smug in the moment.

Reply:
I vote we make it private, keep that data as secure as possible.